backend/tests/tests.go

```diff
@@ -211,21 +211,21 @@ var (
 	resourceDir string
 )
 
-// getTestPort reserves two ports, one for server, one for sample pg instance.
+// getTestPort reserves 3 ports, 1 for server, 2 for sample pg instance.
```

Comment:
4 ports, 2 for server(gRPC and gateway)?

Reply:
@d-bytebase pls confirm, also for getTestPortForEmbeddedPg